### PR TITLE
feat(tutorial): complete step 4

### PR DESCRIPTION
**Pull request by cpandhi**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/components/Info/_info.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.info-section__heading {
+  @include carbon--type-style('heading-03');
```
Comment (kellychurchill) [inline src/components/Info/_info.scss]: This should be `$heading-01`

Comment (cpandhi) [reply src/components/Info/_info.scss]: Oh i know what i did wrong there

Comment (cpandhi) [reply src/components/Info/_info.scss]: Done!

**Comment by kellychurchill** [comment]
👍 Looks good - I have recorded your completion of this step!

**Pull request closed without merging**